lib/python/ensembl/brc4/runnable/process_gff3.py

```diff
@@ -264,16 +270,44 @@ def simpler_gff3(self, gff3_in: TextIO, out_gff_path: Path, out_funcann_path: Pa
                 new_records.append(new_record)
             
             if fail_types and not skip_unrecognized:
-                raise Exception("Unrecognized types found (%s): fail" %
-                                (" ".join(fail_types.keys())))
+                raise Exception(f"Unrecognized types found ({' '.join(fail_types.keys())}): fail")
             
             GFF.write(new_records, gff3_out)
         
         # Write functional annotation
         functional_annotation = self.clean_functional_annotations(functional_annotation)
         print_json(out_funcann_path, functional_annotation)
 
-    def normalize_gene(self, gene: SeqFeature, functional_annotation: List, fail_types: List) -> SeqFeature:
+    def format_mobile_element(self, feat, functional_annotation):
+        """Given a mobile_genetic_element feature, transform it into a transposable_element"""
+        quals = feat.qualifiers
+
+        if feat.type == 'mobile_genetic_element':
+            mobile_element_type = feat.qualifiers.get("mobile_element_type", [])
+            if mobile_element_type:
+                element_type, transposon_type = mobile_element_type[0].split(':')
+                if element_type in ("transposon", "retrotransposon"):
```

Comment:
```suggestion
                if element_type.endswith("transposon"):
```

Reply:
I'd prefer we keep the explicit list, since those are ontology terms (which we might expand in the future if for some reason there are other types or transposons we might accept)